src/mercury_engine_data_structures/common_types.py

```diff
@@ -13,29 +32,33 @@
 
 StrId = CStringRobust("utf-8")
 Char = StaticPaddedString(1, "utf-8")
-Int: construct.FormatField = typing.cast(construct.FormatField, construct.Int32sl)
-UInt: construct.FormatField = typing.cast(construct.FormatField, construct.Int32ul)
-Float: construct.FormatField = typing.cast(construct.FormatField, construct.Float32l)
-CVector2D = construct.Array(2, Float)
-CVector3D = construct.Array(3, Float)
-CVector4D = construct.Array(4, Float)
+Int: FormatField = typing.cast(FormatField, Int32sl)
+UInt: FormatField = typing.cast(FormatField, Int32ul)
+Float: FormatField = typing.cast(FormatField, Float32l)
+CVector2D = Array(2, Float)
+CVector3D = Array(3, Float)
+CVector4D = Array(4, Float)
 
 class VersionAdapter(Adapter):
-    def __init__(self, value: int | str | tuple[int, int, int] | None = None):
```

Comment:
There's a lot of clutter in this file since I refactored it to explicitly import the construct data we use here, but this is the important bit. 1/2